### DISCUSSION TITLE
React On Rails without webpack using HTM

### DISCUSSION THREAD

**Discussion by vccoffey** [shakacode/react_on_rails#1552] [General]
I’m working upgrading to Rails 7 the Import Map way - getting rid of webpack.

I’m interested in using the HTM (Hyperscript Tagged Markup) solution demo’d by DHH here: [Alpha preview: Using React with importmaps on Rails 7 - YouTube](https://www.youtube.com/watch?v=k73LKxim6tw)

React On Rails would still be super helpful in providing view helpers and possibly server side rendering (though we are not using SSR in our app).

Is React On Rails compatible with this non-webpack HTM approach?

**Comment by justin808** [comment]
@vccoffey I'm open to a PR if you want. Or if you want to create a demo app, I'll review it.

**Reply by Judahmeek** [reply]
Why is this question being asked under Shakapacker discussions though?

**Reply by justin808** [reply]
Yes -- please move to React on Rails.

**Reply by vccoffey** [reply]
Yes my mistake this should be in React On Rails
